modules/sno-du-tuning-the-performance-patch.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-far-edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="sno-du-tuning-the-performance-patch_{context}"]
+= Extended Tuned profile configuration
+
+The following example `Tuned` CR extends the `Tuned` profile with additional performance tuning configurations necessary for high-performance workloads.
+
+.Recommended extended tuned profile configuration
+[source,yaml]
+----
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: performance-patch
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+    - data: |
+        [main]
+        summary=Configuration changes profile inherited from performance created tuned
+        include=openshift-node-performance-openshift-node-performance-profile
+        [bootloader]
+        cmdline_crash=nohz_full=2-19,22-39
```

Comment:
It's best to match these cpus with workload partitioning configuration.

With this wp config:  "0-1,52-53", I expect this in tuned:
`cmdline_crash=nohz_full=2-51,54-103`

Reply:
This is not updated yet.

Reply:
updated, PTAL

Reply:
Made these changes.